.github/workflows/unit-test-code-analysis.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+        with:
```

Comment:
This was added to fix an error when running CodeCov in GH Action.
Error is here
```
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS brotli GSS-API HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets
==> GitHub Actions detected.
    Env vars used:
      -> GITHUB_ACTIONS:    true
      -> GITHUB_HEAD_REF:   lock-bootstrap-version
      -> GITHUB_REF:        refs/pull/470/merge
      -> GITHUB_REPOSITORY: awslabs/service-workbench-on-aws
      -> GITHUB_RUN_ID:     807977121
      -> GITHUB_SHA:        f41ac82e088df80b6aea33b828b5bf8629ee34f7
      -> GITHUB_WORKFLOW:   Unit Tests & Code Analysis
->  Issue detecting commit SHA. Please run actions/checkout with fetch-depth > 1 or set to 0
```
https://github.com/awslabs/service-workbench-on-aws/pull/470/checks?check_run_id=2495242442

Recommended fix that was followed
https://github.com/codecov/codecov-action/issues/190#issuecomment-790729633